spec/models/user_spec.rb

```diff
@@ -1,5 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  it { should validate_presence_of(:username)}
+  context 'validations' do
```

Comment:
So here instead of opening up a ```do..end``` block you could just write it inline like so: ``` it { should validate_uniqueness_of(:username).case_insensitive }. You also don't have to write the model validations within a context block. We really only need to use contexts within our feature tests